### DISCUSSION TITLE
[New Feature] Export to Word (`.docx`) with math

### DISCUSSION THREAD

**Discussion by windingwind** [windingwind/zotero-better-notes#823] [Announcements]
See steps to export in #822.

<img width="1050" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/bb0a90b7-455f-4868-bf59-3a6e8e9fb852">

**Comment by s1291** [comment]
@windingwind Thanks for the update
For me, on Windows 11, the export works for some notes. But for larger ones, nothing gets exported, and the note gets duplicated every time I try to export it.

**Reply by windingwind** [reply]
1. make sure you're on the latest BN
2. if you do not use the `embed note links`/`export note links` feature, use the 3rd option (`keep link`) instead of the other two:

<img width="248" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/1e342a15-c2cb-477b-a6e5-6a37a56d676f">

3. report bug with a debug output in a new issue.

**Reply by s1291** [reply]
Issue solved by the new release: [1.1.4-44](https://github.com/windingwind/zotero-better-notes/releases/tag/1.1.4-44)
